build.sbt

```diff
@@ -163,6 +164,31 @@ lazy val `java-customer-registry` = project
     IntegrationTest / PB.protoSources ++= (Compile / PB.protoSources).value
   )
 
+/*
+can't be included since it depends on codegen to be compileable?
+lazy val `java-eventsourced-shopping-cart` = project
+  .in(file("samples/java-eventsourced-shopping-cart"))
+  .dependsOn(sdk)
+  .enablePlugins(AkkaGrpcPlugin, IntegrationTests, LocalDockerImage)
+  .disablePlugins(HeaderPlugin)
+  .settings(
+    name := "java-eventsourced-shopping-cart",
+    libraryDependencies ++= Seq(
+      "ch.qos.logback" % "logback-classic" % LogbackVersion,
+      "ch.qos.logback.contrib" % "logback-json-classic" % LogbackContribVersion,
+      "ch.qos.logback.contrib" % "logback-jackson" % LogbackContribVersion,
+      "org.junit.jupiter" % "junit-jupiter" % JUnitJupiterVersion % IntegrationTest,
+      "net.aichler" % "jupiter-interface" % JupiterKeys.jupiterVersion.value % IntegrationTest
+    ),
+    Compile / akkaGrpcGeneratedLanguages := Seq(AkkaGrpc.Java),
+    Compile / javacOptions ++= Seq("-encoding", "UTF-8", "-source", "11", "-target", "11"),
+    testOptions += Tests.Argument(jupiterTestFramework, "-q", "-v"),
+    inConfig(IntegrationTest)(JupiterPlugin.scopedSettings),
+    IntegrationTest / akkaGrpcGeneratedSources := Seq(AkkaGrpc.Client),
+    IntegrationTest / PB.protoSources ++= (Compile / PB.protoSources).value
+  )
+ */
```

Comment:
I didn't get the intention of this. Was that for local tests? 

In any case, we don't want the project in sbt but in maven and that's already setup to use the current codegen and sdk. So, maven tests should work.

Reply:
I added this to get faster feedback when working on it by including it in the sbt build, but then that applies all header and formatting and whatnot so it turned out it wasn't a good idea.